hwcomposer/hwcomposer.pro

```diff
@@ -29,6 +29,11 @@ HEADERS += hwcomposer_backend_v11.h
 
 QT += core-private compositor-private gui-private platformsupport-private dbus
 
+enable-sensors {
+    QT += sensors
```

Comment:
Just out of curiosity, why do we need sensors in the hwcompositor plugin?

Reply:
We use it in LuneOS for rotation ;)

Reply:
My guess would be for a different compositor stack, to enable rotation. I'm not sure the QPA plugin would really be the right place to put it architecturally, though.

Reply:
We use a different stack in LuneOS v.s. Mer. For our setup this is the most logical place, but we made it optional, so shouldn't impact others that don't enable it :)